appinventor/components/tests/com/google/appinventor/components/runtime/CheckboxTest.java

```diff
@@ -0,0 +1,61 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright © 2017-2018 Massachusetts Institute of Technology, All rights reserved.
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package com.google.appinventor.components.runtime;
+
+
+import com.google.appinventor.components.runtime.shadows.org.osmdroid.tileprovider.modules.ShadowMapTileModuleProviderBase;
+import com.google.appinventor.components.runtime.shadows.org.osmdroid.views.ShadowMapView;
+import org.junit.Before;
+import org.junit.Test;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test class for Switch component.
+ *
+ * @author srlane@mit.edu (Susan Rati Lane)
+ */
+@Config(shadows = {ShadowMapView.class, ShadowMapTileModuleProviderBase.class})
```

Comment:
You shouldn't need these shadows here as they are specific to testing Map-related components.